website/content/docs/connect/config-entries/http-route.mdx

```diff
@@ -34,6 +34,25 @@ to view additional details, including default values.
       - [`Set`](#rules-filters-headers-set): map | no default
     - [`URLRewrite`](#rules-filters-urlrewrite): map | no default
       - [`Path`](#rules-filters-urlrewrite-path): string | no default
+    - [`RetryFilter`](#rules-filters-retryfilter): map
+      - [`NumRetries`](#rules-filters-retryfilter-numretries): number | `1`
+      - [`RetryOnConnectionFailure`](#rules-filters-retryfilter-retryonconnectionfailure): boolean | `false`
+      - [`RetryOn`](#rules-filters-retryfilter-retryon): list of strings
+      - [`RetryOnStatusCodes`](#rules-filters-retryfilter-retryon): list of numbers
+    - [`TimeoutFilter`](#rules-filters-timeoutfilter): map
+      - [`IdleTimeout`](#rules-filters-timeoutfilter): number | `0`
+      - [`RequestTimeout`](#rules-filters-timeoutfilter): number | `0`
+    - [`JWT`](#rules-filters-jwt): map
+         - [`Providers`](#rules-filters-jwt-providers): list
+            - [`Name`](#rules-filters-jwt-providers): string
+            - [`VerifyClaims`](#rules-filters-jwt-providers): map
+               - [`Path`](#rules-filters-jwt-providers): list
+               - [`Value`](#rules-filters-jwt-providers): string
+  - [`ResponseFilters`](#rules-responsefilters)
```

Comment:
```suggestion
  - [`ResponseFilters`](#rules-responsefilters): map
```
@t-eckert - the description in the spec refers to this as a list, but it is a map in the complete configuration